File: documentation-examples-scala/src/scala/dev/typr/foundations/docs/oracle/FloatTypes.scala
package dev.typr.foundations.docs.oracle
import dev.typr.scalafoundations.*
import dev.typr.scalafoundations.data.*



@SuppressWarnings(Array("unused"))
object FloatTypes:
  //start
  val binaryFloat: OracleType[Float] = OracleTypes.binaryFloat
  val binaryDouble: OracleType[Double] = OracleTypes.binaryDouble
  val floatType: OracleType[Double] = OracleTypes.float_(126)  // FLOAT(126)
  //stop

File: documentation-examples-java/src/java/dev/typr/foundations/docs/oracle/FloatTypes.java
package dev.typr.foundations.docs.oracle;

import dev.typr.foundations.OracleType;
import dev.typr.foundations.OracleTypes;

@SuppressWarnings("unused")
public class FloatTypes {
  // start
  OracleType<Float> binaryFloat = OracleTypes.binaryFloat;
  OracleType<Double> binaryDouble = OracleTypes.binaryDouble;
  OracleType<Double> floatType = OracleTypes.float_Of(126); // FLOAT(126)
  // stop
}
